components/tabstrip/tabs.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Tabs
+page_title: TabStrip Tabs
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs
+tags: telerik,blazor,tab,strip,tabstrip,overview
+published: True
+position: 7
+---
+
+# Tabs
+
+You can configure the `TabStripTab` component and its child `TabStrip` instances through the following parameters:
+
+* [Position](#position)
+* [Title](#title)
+* [Visible](#visible)
+* [Disabled](#disabled)
+
+
+## Position
```

Comment:
this one is at the tabstrip level, the rest are tab-specific, differentiate that, or, better yet split into two articles

Reply:
Updated - split in two articles.